src/ts/replayprotection/bitflip.ts

```diff
@@ -12,14 +12,17 @@ import BN from "bn.js";
  * does not matter).
  */
 export class BitFlip extends ReplayProtectionAuthority {
```

Comment:
Should probably bet call BitFlipReplayProtection, and likewise for the MultiNonce

Reply:
done